src/main/java/seedu/lifeasier/ui/Ui.java

```diff
@@ -582,4 +582,12 @@ public void showInvalidRecurrencesError() {
         System.out.println(colourTextRed("Recurrences must be a positive integer!"));
         printSeparator();
     }
+
+    public void showDuplicateTaskError(String type) {
+        System.out.println("This " + type + " or something very similar already exists in your schedule!");
+    }
+
+    public void showPastTaskError(String type) {
+        System.out.println("The timing of this " + type + " is already in the past!");
+    }
```

Comment:
You might want to color these red

Reply:
I wanted to wait until we merged my other commits, so that I can add the blank lines and separators too. Thanks for pointing it out though!